lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -254,15 +253,22 @@ def paths_for_attribute(attribute, include_dirs = false)
       # Matches the given patterns to the file present in the root of the path
       # list.
       #
-      # @param [Array<String>] patterns
+      # @param  [Array<String>] patterns
       #         The patterns to expand.
       #
-      # @param  [String] dir_pattern
+      # @param  [Hash] options
+      #         The options to use to expand the patterns to file paths.
+      #
+      # @option options [String] :dir_pattern
       #         The pattern to add to directories.
       #
-      # @param  [Array<String>] exclude_patterns
+      # @option options [Array<String>] :exclude_patterns
```

Comment:
Probably same for `:dir_pattern` above. (I believe that's from @mrackwitz 's commit?)